integration_test/package.json

```diff
@@ -6,8 +6,12 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
-    "test:android": "yarn cavy run-android",
-    "test:ios": "yarn cavy run-ios",
+    "stop-test:android": "adb shell am force-stop com.bitmovin.player.reactnative.integrationtest",
+    "stop-test:ios": "xcrun simctl list devices available -e -j | jq --raw-output '.devices.[].[] | select( .state == \"Booted\" and (.deviceTypeIdentifier | contains(\"iPhone\"))).udid' | xargs -I UDID xcrun simctl terminate UDID com.bitmovin.player.reactnative.IntegrationTest 2> /dev/null; exit 0",
```

Comment:
These will stop already running instance of the integration test app to prevent it executing tests while rebuilding.